BridgeAppSDK.xcodeproj/project.pbxproj

```diff
@@ -186,6 +185,7 @@
 		FF92A6791DBFF13700614D1F /* Instructions.html in Resources */ = {isa = PBXBuildFile; fileRef = FF92A67B1DBFF13700614D1F /* Instructions.html */; };
 		FF92A67F1DBFF14000614D1F /* Withdrawl.html in Resources */ = {isa = PBXBuildFile; fileRef = FF92A6811DBFF14000614D1F /* Withdrawl.html */; };
```

Comment:
apparently this has been like this all along... not important to fix unless you feel like it